realm/kotlin-extensions/src/androidTestObjectServer/kotlin/io/realm/kotlin/KotlinSyncedRealmTests.kt

```diff
@@ -33,11 +30,13 @@ class KotlinSyncedRealmTests {
         realm.close()
     }
 
+    @Ignore("FIXME")
```

Comment:
Like mentioned other PR comments can we use one issue to track all the ignored/disabled test/code